velox/expression/LambdaExpr.cpp

```diff
@@ -132,6 +132,13 @@ class ExprCallable : public Callable {
 
 } // namespace
 
+void LambdaExpr::computeDistinctFields() {
+  SpecialForm::computeDistinctFields();
```

Comment:
Should we call this after filling in distinctFields_? Otherwise, multiplyReferencedFields_ won't be updated, no?

Can we add a check that distinctFields_ is empty?

Reply:
good point, lemme call mergeFields() while adding the new fields in the specialized implementations, that way we can ensure the parent implementation is at the top while ensuring we also update the multiply referenced fields